[["file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/http-kernel/Exception/TooManyRequestsHttpException.php",{"_uri":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/http-kernel/Exception/TooManyRequestsHttpException.php","_root":{"kind":4096,"name":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/http-kernel/Exception/TooManyRequestsHttpException.php","location":{"uriHash":711728639,"range":{"start":{"line":0,"character":0},"end":{"line":36,"character":0}}},"children":[{"kind":512,"name":"Symfony\\Component\\HttpKernel\\Exception","location":{"uriHash":711728639,"range":{"start":{"line":11,"character":0},"end":{"line":11,"character":49}}}},{"kind":1,"name":"Symfony\\Component\\HttpKernel\\Exception\\TooManyRequestsHttpException","location":{"uriHash":711728639,"range":{"start":{"line":18,"character":0},"end":{"line":35,"character":1}}},"children":[{"kind":32,"name":"__construct","location":{"uriHash":711728639,"range":{"start":{"line":26,"character":4},"end":{"line":34,"character":5}}},"doc":{"description":"","type":""},"modifiers":1,"type":"","children":[{"kind":128,"name":"$retryAfter","location":{"uriHash":711728639,"range":{"start":{"line":26,"character":32},"end":{"line":26,"character":50}}},"doc":{"description":"The number of seconds or HTTP-date after which the request may be retried","type":"int|string"},"value":"null","scope":"__construct"},{"kind":128,"name":"$message","location":{"uriHash":711728639,"range":{"start":{"line":26,"character":52},"end":{"line":26,"character":67}}},"doc":{"description":"The internal exception message","type":"string"},"value":"null","scope":"__construct"},{"kind":128,"name":"$previous","location":{"uriHash":711728639,"range":{"start":{"line":26,"character":69},"end":{"line":26,"character":96}}},"type":"Exception","doc":{"description":"The previous exception","type":"Exception"},"value":"null","scope":"__construct"},{"kind":128,"name":"$code","location":{"uriHash":711728639,"range":{"start":{"line":26,"character":98},"end":{"line":26,"character":107}}},"doc":{"description":"The internal exception code","type":"int"},"value":"0","scope":"__construct"}],"scope":"Symfony\\Component\\HttpKernel\\Exception\\TooManyRequestsHttpException"}],"associated":[{"kind":1,"name":"Symfony\\Component\\HttpKernel\\Exception\\HttpException"}],"modifiers":0}]},"_hash":711728639}]]